utils/releaser/src/ReleaseWorker/ReleaseCandidate/UpdateChangelogReleaseWorker.php

```diff
@@ -30,6 +30,10 @@ public function work(
         Version $version,
         string $initialBranchName = AbstractShopsysReleaseWorker::MAIN_BRANCH_NAME,
     ): void {
+        $this->symfonyStyle->note(sprintf('You need to push the "%s" branch manually', $this->currentBranchName));
```

Comment:
is it necessary to push the RC branch before? 
changelog is generated from a base branch anyway, or?

Reply:
Oh, there is other way to correct target, I did that